src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/HistoricalIncidentResponse.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.digital.hmpps.incidentreporting.jpa
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.OneToMany
+import jakarta.persistence.OneToOne
+import java.time.LocalDateTime
+
+@Entity
+class HistoricalIncidentResponse(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  val incident: IncidentReport,
+
+  val dataPointKey: String,
```

Comment:
minor point on naming… is “data point” a little too generic? as in, can data points be grouped together into clusters of some sort? tbh, at this point we don’t know what we’re making so i have no better terminology

Reply:
Yes happy to change - just don't know to what!